ififuncs.py

```diff
@@ -899,7 +899,16 @@ def get_script_version(scriptname):
     current_dir = os.getcwd()
     home = os.path.expanduser("~/")
     os.chdir(home)
-    if os.path.isdir('ifigit/ifiscripts'):
+    pypi = subprocess.check_output([
+        'pip', 'show', 'ifiscripts'
```

Comment:
Won't this fail on mac as you don't have a pip3 in here? If pip2 is already installed, then just saying `pip` will be calling pip2 instead of pip3, so you'll either get an error or an older version returned.
Might need to make two command lists and perform an operating system check and launch the pip3 list if mac/darwin, launch pip if windows. Might need something for linux as well. There's operating system checks in ifiscripts as well.

Reply:
for example in pseudo code, it's ' check if mac or windows, if mac, pip_cmd='[pip3, show, ifiscripts], if windows, pip_cmd=[pip, show, ifiscripts], then you can just do subprocess.check_output(pip_cmd). Or you can make pip a variable as well and have one command list. Many options!

Reply:
Oh sorry I just realise this - I don't why `pip` and `pip3` are doing the same work in the same place

Reply:
fixed:
```
pypi = ''
try:
    if platform.system() == 'Windows':
        pypi_cmd = ['pip', 'show', 'ifiscripts']
    else:
        pypi_cmd = ['pip3', 'show', 'ifiscripts']
    pypi = subprocess.check_output(pypi_cmd)
    pypi = pypi.decode()
except Exception as e:
    print(e)
if pypi:
......
```